tensorflow_model_optimization/python/examples/quantization_with_sparsity/keras/mnist_cnn.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```

Comment:
It'd be good to consider the discussion in https://github.com/tensorflow/model-optimization/pull/562 together with this.

Reply:
Thanks for point it out. Did we have conclusion regarding example scripts?

Reply:
I think it's appropriate to not have the scripts in this case and substitute them with e2e unit tests (that don't take too long to run) and other documentation.